### PR TITLE
Allow setting ramdisk SSH key for debugging

### PR DESCRIPTION
**Pull request by dtantsur**

### CODE REVIEW

**Comment by dtantsur** [comment]
/hold

I need to figure out how to test it

**Comment by derekhiggins** [comment]
Can you rebase and I'll test this out, looks like we'll also now need the key added to inspector.extra_kernel_params in configure-ironic.sh.

**Comment by derekhiggins** [comment]
Can be used by installer with something like this
https://github.com/openshift/installer/pull/3963

File: runhttpd.sh [hunk 1]
```diff
@@ -15,7 +23,10 @@ cp /tmp/dualboot.ipxe /shared/html/dualboot.ipxe
 cp /tmp/uefi_esp.img /shared/html/uefi_esp.img
 
 # Use configured values
-sed -i -e s/IRONIC_IP/${IRONIC_URL_HOST}/g -e s/HTTP_PORT/${HTTP_PORT}/g /shared/html/inspector.ipxe
+sed -i -e s/IRONIC_IP/${IRONIC_URL_HOST}/g \
+    -e s/HTTP_PORT/${HTTP_PORT}/g \
+    -e "s/KERNEL_PARAMS/${KERNEL_PARAMS:-}/g" \
```
Comment (dhellmann) [inline runhttpd.sh:28]: Do we have to worry about KERNEL_PARAMS having a `/` in it?

**Pull request closed without merging**